lib/routes/account.js

```diff
@@ -258,7 +309,15 @@ module.exports = function (
             sessionToken: isA.string().regex(HEX_STRING).required(),
             keyFetchToken: isA.string().regex(HEX_STRING).optional(),
             verified: isA.boolean().required(),
-            authAt: isA.number().integer()
+            authAt: isA.number().integer(),
+            device: isA.object({
+              id: isA.string().length(32).regex(HEX_STRING).required(),
+              name: isA.string().max(255).optional().allow(''),
+              type: isA.string().max(16).optional(),
+              pushCallback: isA.string().uri({ scheme: 'https' }).max(255).optional().allow(''),
+              pushPublicKey: isA.string().length(64).regex(HEX_STRING).optional().allow('')
```

Comment:
IIUC, devices that can't/don't support a push endpoint will have to explicitly specify an empty string in these fields.  I'm not *opposed* to this, but it does seem like extra work when we could allow them to just omit the field entirely.  Any particular reason to require them?